src/providers/journal/__tests__/journal.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { JournalProvider } from '../journal';
+import { remoteEnvironmentMock, localEnvironmentMock } from '../../app-config/__mocks__/environment.mock';
+import { AppConfigProvider } from '../../app-config/app-config';
+import { AuthenticationProvider } from '../../authentication/authentication';
+import { AuthenticationProviderMock } from '../../authentication/__mocks__/authentication.mock';
+
+describe('JournalProvider', () => {
+  describe('getJournal', () => {
+
+    let journalProvider;
+    let httpMock;
+    let authProviderMock;
+    let appConfig;
+    
+    beforeEach(() => {
+      TestBed.configureTestingModule({
+        imports: [
+          HttpClientTestingModule
+        ],
+        providers: [
+          JournalProvider,
+          { provide: AppConfigProvider, useClass: AppConfigProvider, environmentFile: remoteEnvironmentMock },
```

Comment:
Should we not be using the mock here for the AppConfigProvider.

You then don't have to mock the employee id later on in the test.